bin/templates/cordova/lib/check_reqs.js

```diff
@@ -393,11 +393,13 @@ module.exports.run = function () {
         console.log('ANDROID_SDK_ROOT=' + process.env['ANDROID_SDK_ROOT'] + ' (recommended setting)');
         console.log('ANDROID_HOME=' + process.env['ANDROID_HOME'] + ' (DEPRECATED)');
 
-        if (!String(values[0]).startsWith('1.8.')) {
+        if (values[0] === undefined) {
```

Comment:
Wouldn't this, theoretically, allow JDK <8, too? They would, e.g., start with 1.6 for a JDK6. We should maybe use something like [this](https://stackoverflow.com/a/1096159/1902598) in order to check for >= 52

Reply:
Agreed, we should still throw an error for JDK < 8. We might be able to use semver. We already depend on it IIRC.

Reply:
The problem is that in Java <=1.8 it’s labeled as `1.x` while Java 9 and newer dropped the `1.`. This complicates the check a little bit.

Reply:
Well, using semver it would still only entail checking that we satisfy `>=1.8`.

Reply:
Indeed, 14 > 1.8, so this could work/be a sufficient check.

Reply:
With `semver` the following should work:
```suggestion
        if (!semver.satisfies(semver.coerce(values[0]), '>= 1.8')) {
```